### PR TITLE
chore: add elyra pipeline on telco

### PR DESCRIPTION
**Pull request by ubabe53**
Here's the pipeline, let me know if everything it is okay! @gmartinsribeiro

### CODE REVIEW

File: demo/Telecommunications/5 - Telco Anomaly Detection Elyra/check_data_quality.ipynb [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "from ydata_quality import DataQuality\n",
+    "import pandas as pd\n",
+    "\n",
+    "#Load in the data\n",
+    "df = pd.read_csv('data.csv')\n",
```
Comment (gmartinsribeiro) [inline demo/Telecommunications/5 - Telco Anomaly Detection Elyra/check_data_quality.ipynb:80]: are we using pandas to read the data????

File: demo/Telecommunications/5 - Telco Anomaly Detection Elyra/preprocess_data.ipynb [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6c1b1fb8-007f-4087-9e6d-740c99c4a12e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pandas import read_csv\n",
+    "\n",
+    "data = read_csv('data.csv')"
```
Comment (gmartinsribeiro) [inline demo/Telecommunications/5 - Telco Anomaly Detection Elyra/preprocess_data.ipynb:12]: again?

**Comment by gmartinsribeiro** [comment]
Something seems weird on the pipelines... How are we "transferring" information between components of the pipeline? We should be using the Artifacts but it seems we're writing and reading datasets...

File: demo/Telecommunications/5 - Telco Anomaly Detection Elyra/read_data.ipynb [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+   "execution_count": 22,
+   "id": "eae29b9a-865a-4fe3-be4d-e5f4bb187453",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Output the data. In the elyra pipeline, you need to output the data in order to let it be available for the next steps.\n",
+    "final_df = data.to_pandas()\n",
+    "final_df.to_csv('data.csv', index=False)"
```
Comment (gmartinsribeiro) [inline demo/Telecommunications/5 - Telco Anomaly Detection Elyra/read_data.ipynb:67]: shouldn't we be using our LocalConnector to write the file to the output?

Comment (gmartinsribeiro) [reply demo/Telecommunications/5 - Telco Anomaly Detection Elyra/read_data.ipynb:67]: Yup, that's right! If we have our own way of dealing with the data flow, let's use it.

Comment (ubabe53) [reply demo/Telecommunications/5 - Telco Anomaly Detection Elyra/read_data.ipynb:67]: Ok. I will change the notebooks and use just the connectors.

File: demo/Telecommunications/5 - Telco Anomaly Detection Elyra/read_data.ipynb [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+   "cell_type": "code",
+   "execution_count": 24,
+   "id": "24d5a3d7-c058-4d04-99f5-feec8d5a4838",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Here we create the visualization of the table. This is the metadata that kubeflows need to show some lines of the dataset.\n",
+    "import json\n",
+    "\n",
+    "schema = [{\"name\": c, \"type\": \"NUMBER\" } for c in final_df.columns]\n",
```
[the remark this answers is not in the document]
Comment (ubabe53) [reply demo/Telecommunications/5 - Telco Anomaly Detection Elyra/read_data.ipynb:80]: Here is the "schema" that is needed for a table visualization. I followed the one on some examples of kubeflow, but I think is just the list of columns so I'm not sure we can use the YData Metadata. Need to check

[the remark this answers is not in the document]
Comment (fabclmnt) [reply demo/Telecommunications/5 - Telco Anomaly Detection Elyra/read_data.ipynb:80]: Yes I understand. But in order to get the variable types you can get those from our metadata that's my point. You can use the logic from kubeflow examples but you've to adapt it.

[the remark this answers is not in the document]
Comment (ubabe53) [reply demo/Telecommunications/5 - Telco Anomaly Detection Elyra/read_data.ipynb]: at the end I checked, and just the list of names of columns was needed. So I just passed that!

**Comment by fabclmnt** [comment]
@ubabe53 where can I fins this pipeline in the platform? Can you please share it with me.

**Comment by ubabe53** [comment]
@fabclmnt  I shared my workspace with you (dev). You can find the notebooks in the telco_pipeline_2 lab !

**Comment by gmartinsribeiro** [comment]
@ubabe53 Is this ready or has pending changes?

**Comment by ubabe53** [comment]
@gmartinsribeiro this has not pending changes, is ready. If you give me the okay I can push it

**Comment by gmartinsribeiro** [comment]
Looks good to me :)

**Pull request merged**